test-runtimes/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Exclude lockfiles for packaging tests to reduce file churn
+/*/package-lock.json
```

Comment:
Is there a preference for this over `.nvmrc` for each project and checking in `package-lock.json`?

Reply:
`.nvcmrc` for each project would mean many versions of Node, which we don't need... yet